### ISSUE TITLE
Apply personalization color on IdsTab color-variant: module

### ISSUE THREAD

**Issue by kamilbauer** [infor-design/enterprise-wc#2617]
**Describe the bug**
Using the [personalize component](https://main.wc.design.infor.com/ids-personalize/example.html) changes color on regular tabs, but does not work for color-variant: module.

![image](https://github.com/user-attachments/assets/a736ce23-c8e0-4d23-bf07-a36fd30de869)

**To Reproduce**

1. Add personalize component
2. Add ids-tabs color-variant: module
3. Change color using the personalize component

**Expected behavior**
Color should be consistent with the rest of the application, like in regular tabs.
![image](https://github.com/user-attachments/assets/afbaeadb-99e2-4989-b90f-e2dac0c5bd55)

**Version**
1.3.0

**Screenshots**
![image](https://github.com/user-attachments/assets/a736ce23-c8e0-4d23-bf07-a36fd30de869)

**Platform**
 - Infor Application/Team Name: Ids Web Components
 - Device: PC
 - OS Version: Windows 11
 - Browser Name: Chrome
 - Browser Version: 126.0.6478.128

**Comment by tmcconechy** [comment]
What we personalize is a bit up in the air right now. We might not be doing this on tabs module going forward. Will discuss with the team.

**Comment by echodunchie** [comment]
Hi @tmcconechy, is the color variant for tabs (header) also part of this decision?

**Comment by echodunchiee** [comment]
Hi @tmcconechy, is the color variant for tabs (header) also part of this decision?

**Comment by tmcconechy** [comment]
Yes, The question is going to be a) what if anything we customize the color on b) if we do it at all?
